libnmstate/netapplier.py

```diff
@@ -109,7 +110,13 @@ def _apply_ifaces_state(plugins, net_state, verify_change, save_to_disk):
         plugin.apply_changes(net_state, save_to_disk)
     verified = False
     if verify_change:
-        for _ in range(VERIFY_RETRY_TIMEOUT):
+        if _net_state_contains_sriov_interface(net_state):
+            # If SR-IOV is present, the verification timeout is being increased
+            # to avoid timeouts due to slow drivers like i40e.
+            verify_retry = VERIFY_RETRY_TIMEOUT * 4
```

Comment:
`4` should be a constant. And why 4?

Reply:
We are retrying up to 4 times more. Yes, I will set it as a constant. 20 seconds should be enough time.

Reply:
Fixed!